libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -132,33 +148,32 @@ char ofEasyCam::getTranslationKey() const{
 }
 
 //----------------------------------------
-void ofEasyCam::enableMouseInput(){
-	if(!bMouseInputEnabled && events){
-		listeners.push_back(events->update.newListener(this, &ofEasyCam::update));
-		listeners.push_back(events->mouseDragged.newListener(this, &ofEasyCam::mouseDragged));
-		listeners.push_back(events->mousePressed.newListener(this, &ofEasyCam::mousePressed));
-		listeners.push_back(events->mouseReleased.newListener(this, &ofEasyCam::mouseReleased));
-		listeners.push_back(events->mouseScrolled.newListener(this, &ofEasyCam::mouseScrolled));
+void ofEasyCam::enableMouseInput(bool e){
```

Comment:
why this? we don't use this pattern anywhere else, things are called either set*(bool) or enable/disable but we don't use enable(bool) anywhere so i would avoid having it here